packages/cozy-intent/readme.md

```diff
@@ -0,0 +1,122 @@
+# Cozy-intent
+
+## Concept
+
+This library allows a react-native and a react app running in a webview to communicate in unidirectional and/or bidirectional fashions. The library is following declarative paradigms and is fully strictly typed. An emphasis has been put on low configuration needs and ease of use.
+
+In a nutshell, one end says what it wants to an other end, and the other end handles it any way it wants. For more information on the underlying systems of this library, see https://github.com/alesgenova/post-me.
+
+## Installation
+
+The library needs to be installed both in the parent and the children application.
+
+- `yarn add cozy-webview-intent`
+
+## API
+
+### Nomenclature
+
+- `event`: simple function that sends an event to the listening messenger.
+- `intent`: high-level abstraction intended to describe the whole flow of events/methods and the actual result of those calls in the application.
+- `messenger`: class implementing `post-me` messenger paradigm. Messengers are environment bound (DOM, React-native, worker, iframe, etc). They send and listen methods and events to other connected messengers.
+- `method`: messenger function that accept simple arguments to be handled by the listening messenger.
```

Comment:
Same comment here. If you want to refer to the type of arg (i.e non complex objects), maybe you can use `primitive` instead. But there seems not to be the case?
```suggestion
- `method`: messenger function that accept arguments to be handled by the listening messenger.
```

Reply:
Good point because I'm not even sure myself what arguments are valid (obviously string-like is ok, like boolean, numbers.. but objects? functions? I need to test, post-me is not clear on that)